eng/packaging.props

```diff
@@ -72,5 +72,8 @@
           Include="$(PkgDir)useSharedDesignerContext.txt">
         <SkipPackageFileCheck>true</SkipPackageFileCheck>
     </File>
+
+    <!-- Make sure that NETStandard.Library package never gets added as a dependency for .NET 4.6.1 -->
+    <SuppressMetaPackage Include="NETStandard.Library" TargetFramework="net461" />
```

Comment:
Why only for net461? What about potential other netframework versions for which we might ship an asset in the future?

Reply:
In that case we would simply just add other SuppressMetaPackage items which define  other TargetFramework metadata. For now we just needed this for net461 and I highly doubt that in the future we decide to provide new assets on packages that are from .NET 462- .NET 4.8 so I don't think we have to worry about that.